### PR TITLE
Update ASM to 9.3 in Open Liberty

### LINKED ISSUE #21146: Upgrade ASM to 9.3 in OL
User story:
"As the Liberty development team, we need to upgrade to ASM 9.3 in order to support Java 19."

### PR DESCRIPTION
**Pull request by gjwatts**
Fixes #21146

### CODE REVIEW

**Comment by gjwatts** [comment]
#build

**Comment by gjwatts** [comment]
#build

**Comment by gjwatts** [comment]
#build

**Comment by gjwatts** [comment]
Only one unrelated known UI build break hit.

**Review by Channyboy — APPROVED** [review]
lgtm

**Review by pgunapal — APPROVED** [review]
The request probe changes, LGTM!

File: dev/com.ibm.ws.logging.osgi/src/com/ibm/ws/logging/internal/osgi/stackjoiner/bci/AddVersionFieldClassAdapter.java [hunk 1]
```diff
@@ -15,6 +15,8 @@
 import org.objectweb.asm.Opcodes;
 import org.objectweb.asm.Type;
 
+import io.openliberty.asm.ASMHelper;
```
Comment (nmittles) [inline dev/com.ibm.ws.logging.osgi/src/com/ibm/ws/logging/internal/osgi/stackjoiner/bci/AddVersionFieldClassAdapter.java:18]: Update copyright header.

**Review by nmittles — APPROVED** [review]
Approved.  I added single comment for updating a copyright header date, that can either be taken care of in this PR or included in a future PR.

File: dev/com.ibm.ws.logging.osgi/src/com/ibm/ws/logging/internal/osgi/stackjoiner/bci/AddVersionFieldClassAdapter.java (hunk 1 above)
Comment (gjwatts) [reply dev/com.ibm.ws.logging.osgi/src/com/ibm/ws/logging/internal/osgi/stackjoiner/bci/AddVersionFieldClassAdapter.java:18]: Thanks, fixed.

**Comment by gjwatts** [comment]
Force push was just to update copyright header, no other changes made.

**Comment by gjwatts** [comment]
#libbybot

**Comment by gjwatts** [comment]
Merging

**Pull request merged**